html-forms/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+<title> html-forms</title>
+</head>
+<body>
+<h1>Coding Form!</h1>
+  <form>
+    <div>
+ <label for='Email'>Email:</label><br>
+    </div>
+   <input type='text' id='Email' name='Email'/>
+  </form>
+
+<br>
+
+<form>
```

Comment:
You should only have one `form` element encapsulating all the form control elements, not an individual `form` element for each form control element.